code/game/objects/effects/effect_system.dm

```diff
@@ -206,6 +213,18 @@ steam.start() -- spawns the effect
 		return 0
 	return 1
 
+
+// Fades out the smoke smoothly using it's alpha variable.
+/obj/effect/effect/smoke/proc/fade_out(var/frames = 16)
+	if(!alpha) return //already transparent
```

Comment:
This may be a stupid and small thing, but according to contributing guidelines one-liners like these should be replaced with proper writing.